runtime/parachains/src/paras.rs

```diff
@@ -884,6 +884,22 @@ impl<T: Config> Pallet<T> {
 		Ok(())
 	}
 
+	#[cfg(feature = "runtime-benchmarks")]
+	pub(crate) fn initialize_para_now(id: ParaId, genesis: ParaGenesisArgs) -> DispatchResult {
+		// first queue this para actions..
+		let _ = Self::schedule_para_initialize(id, genesis)?;
+
+		// .. and immediately apply them.
+		Self::apply_actions_queue(Self::scheduled_session());
+
+		// ensure it has become a para.
+		ensure!(
+			ParaLifecycles::<T>::get(id) == Some(ParaLifecycle::Parachain),
+			"Parachain not created properly"
+		);
+		Ok(())
+	}
```

Comment:
I think this is a rather crude way of doing this. Can we initialize the parachain in `register_parachain_with_balance` and then manually move 2 sessions, removing this code from here?

Reply:
moving forward two sessions in a realistic way requires bringing in a helper function like https://github.com/paritytech/polkadot/blob/d16547d930108945fd4d188677d92efbe6f0be06/runtime/parachains/src/hrmp.rs#L1386. Are you sure this is worth it? I have second doubts about it.

Reply:
You know what, on the second thought, with #4457 I think it would be better if I take care of it later.

Reply:
As promised: https://github.com/paritytech/polkadot/pull/4934